manifests/params.pp

```diff
@@ -26,4 +26,5 @@
   Stdlib::HTTPSUrl $candlepin_url = "https://${candlepin_host}:${candlepin_port}/candlepin",
   String[1] $candlepin_client_keypair_group = 'foreman',
   String[1] $postgresql_evr_package = $katello::globals::postgresql_evr_package,
+  String[1] $meta_package = $katello::globals::meta_package,
```

Comment:
Do we need to expose this? If we limit it to globals then the installer can set this via Hiera parameters. I don't see a use case (at least in the short term) where the user would want to override this.

It may not be clear today, but it used to have a parameter (until https://github.com/theforeman/puppet-katello/commit/188a71ff52cd3065492b90be73b12f365b021e5a) that was controlled by the installer.

The long term idea was that you have:

* `globals.pp` is set by the installer
* `params.pp` gets exposed to the user via installer parameters
* `application.pp` and `candlepin.pp` are exposed instead of `init.pp` to allow installation of **only** Candlepin or Foreman.

Then we would expose `foreman_proxy_content` as `--content-*` instead.

An intermediate step was https://github.com/theforeman/puppet-katello/pull/209 which introduced manage flags for each component.

Or we can take the easy solution here and inline it. We should do the same with `postgresql_evr_package` which used to have EL7/EL8 logic. When we dropped EL7 support, there's little benefit to keeping it in globals anymore.

Reply:
Yeah, I don't particularly care where it's defined as long as it can be overridden via installer/hiera. I'll play around how to minify this.

Reply:
do you have a preference? both ways "work", but I find inlining into `params.pp` nicer as in `init.pp` we never touch `katello::globals` directly, so `package { $katello::globals::meta_package … }` reads odd to me (but works as it's loaded by `katello::params`)